app/controllers/concerns/searchable.rb

```diff
@@ -52,4 +48,62 @@ def date_parameters_to_datetime(date_parameters)
   rescue ArgumentError
     nil
   end
+
+private
+
+  def page
+    params[:page] || 1
+  end
+
+  def search_options
+    options = params.permit(:q, from_date: %i[day month year], to_date: %i[day month year]).merge({
+      sort_key: sort_key,
+      sort_dir: sort_dir,
+    })
+
+    options[:verb] = verb if verb.present?
+    options[:adjective] = adjective if adjective.present?
```

Comment:
Not sure how I feel about this being here, as it's specific to a subset of the pages.

Reply:
True. I hadn't thought too closely about it, good to raise it though. I think in time we'll want to separate out `searchable` from the date filter as conceptually they're separate things and some pages use the latter but not the former. I've added a todo because I think it's slightly separate from this PR and we might want to wait a bit anyway. Good spot though 👀